### ISSUE TITLE
Menu Tabs/Items in selected state are not following High contrast standards.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#334]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast standards before launching the application.
1. Launch Xamarin Workbooks application.
2. Navigate among the menu tabs (File/Tools/Run/Help) in collapsed state(Menu tabs under selection) and observe the menu items(under menu tabs) while they are in selected state.
3. Verify the menu items/menu tabs in both High contrast modes.
### Expected Behavior
Should follow selected text High contrast standards
### Actual Behavior
Font color is white in High contrast black and Black in High contrast white. All menu items and menu tabs are hardly visible in selected state in both modes of high contrast.
###User Impact
As the text is hardly visible, low vision user will not able see the text.
###Screenshot
![2 menu items under selection hcb](https://user-images.githubusercontent.com/22956130/40721241-902239c6-6436-11e8-8499-83c5bf465a4a.png)
![2 menu items under selection hcw](https://user-images.githubusercontent.com/22956130/40721242-90521ef2-6436-11e8-83cf-0794c489e9c2.png)
![2 menu tabs under selection in hcb](https://user-images.githubusercontent.com/22956130/40721243-907bc0fe-6436-11e8-9dc8-2282c20ab33d.png)
![2 menu tabs under selection in hcw](https://user-images.githubusercontent.com/22956130/40721244-90a54f5a-6436-11e8-913c-8454f09e54bd.png)
![2 normal mode menu tabs and items](https://user-images.githubusercontent.com/22956130/40721245-90d15744-6436-11e8-9fa6-f139c998f56f.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.4.3

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
